src/modal/Modal.jsx

```diff
@@ -64,14 +64,21 @@ class Modal extends React.Component {
         this.props.onClose && this.props.onClose(event);
     }
 
+    handleCloseIconKeyDown = (event) => {
```

Comment:
Can we name this method as `handleCloseOnKeyDown`

Reply:
Actually, I thought this would be more specific as we're talking about keydown on close icon.

Reply:
Okay! No problem! Will modify later if we can think of some other name.